### DISCUSSION TITLE
[Wiki] Update Tips and Tricks to quickly refresh yabai config

### DISCUSSION THREAD

**Discussion by manugildev** [asmvik/yabai#1866] [General]
Currently, on the [Tips and Tricks](https://github.com/koekeishiya/yabai/wiki/Tips-and-tricks#quickly-restart-the-yabai-launch-agent) page in the wiki it says.

> [Quickly restart the yabai launch agent](https://github.com/koekeishiya/yabai/wiki/Tips-and-tricks#quickly-restart-the-yabai-launch-agent)
> When running as a service, the following command can be used:
> 
> ``` bash
> yabai --restart-service
> 
> # e.g. bind to key in skhd:
> # ctrl + alt + cmd - r : yabai --restart-service"
> ```

However, --restatart-service kicks out the brew command that can be slow at times. Sometimes it even takes 5-10 seconds to restart the service.

The alternative is to refresh the `.yabairc` file only which is way faster:

`source ~/.yabairc`

I suggest to add this option to the Wiki, since most of the times users do not want to restart the whole service, but just to reload their config, for example when a new display is added.

``` bash
multiple_display_setup=$(yabai -m query --displays | jq -r 'length == 2')
if [ $(yabai -m query --displays | jq -r 'length == 2') != false ]; then
	yabai -m rule --add app='^Code$' display=2
else
	yabai -m rule --add app='^Code$' space=4
fi

yabai -m signal --add event=display_added action="source ~/.yabairc"
yabai -m signal --add event=display_removed action="source ~/.yabairc"
```

**Comment by gldtn** [comment]
How can I use the command above to simply reload yabairc without an specific event, just to reload the changes in the config?
